pyhap/accessory_driver.py

```diff
@@ -524,14 +551,12 @@ def pair(self, client_uuid, client_public):
         :return: Whether the pairing is successful.
         :rtype: bool
         """
-        # TODO: Adding a client is a change in the acc. configuration. Then, should we
-        # let the accessory call config_changed, which will persist and update mDNS?
-        # See also unpair.
         logger.info("Paired with %s.", client_uuid)
         self.state.add_paired_client(client_uuid, client_public)
-        self.persist()
+        self.async_persist()
         return True
 
+    @callback
     def unpair(self, client_uuid):
```

Comment:
I didn't rename this to `async_unpair` since it may be overridden in a child class